### PR TITLE
[9.0] Move the account_payment_mode ir.rule in account_payment_mode module

### PR DESCRIPTION
**Pull request by yvaucher**
In order to have it in the proper module

### CODE REVIEW

File: account_payment_order/security/payment_security.xml [hunk 1]
```diff
@@ -14,13 +14,6 @@
...
 
-
-<record id="account_payment_mode_company_rule" model="ir.rule">
-    <field name="name">Payment mode multi-company rule</field>
-    <field name="model_id" ref="model_account_payment_mode"/>
-    <field name="domain_force">['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
-</record>
-
 <!--
 <record id="account_payment_order_company_rule" model="ir.rule">
```
Comment (pedrobaeza) [inline account_payment_order/security/payment_security.xml:18]: @alexis-via, why are these rules commented?

**Comment by pedrobaeza** [comment]
:+1:

**Comment by StefanRijnhart** [comment]
:+1:

**Comment by pedrobaeza** [comment]
@alexis-via, can you say about the commented rules?

**Comment by yvaucher** [comment]
Can we merge this as commented rules are not in scope of this PR ?

**Comment by sbidoul** [comment]
:+1:

**Comment by alexis-via** [comment]
@pedrobaeza I think it's a mistake  ; I guess I commented the ir.rule during the development and I forgot to uncomment them at the end.

**Comment by yvaucher** [comment]
ok then https://github.com/OCA/bank-payment/pull/301

**Pull request merged**